package com.neueda.learning.dto;

import java.math.BigDecimal;

public class AccountResponseDTO {

    private int id;
    private String name;
    private BigDecimal balance;

    public AccountResponseDTO() {
    }

    public AccountResponseDTO(int id, String name, BigDecimal balance) {
        this.id = id;
        this.name = name;
        this.balance = balance;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public BigDecimal getBalance() {
        return balance;
    }

    public void setBalance(BigDecimal balance) {
        this.balance = balance;
    }
}